xbmc/guilib/FFmpegImage.cpp

```diff
@@ -745,7 +746,7 @@ std::shared_ptr<Frame> CFFmpegImage::ReadFrame()
   if (avframe == nullptr)
     return nullptr;
   std::shared_ptr<Frame> frame(new Frame());
-  frame->m_delay = (unsigned int)avframe->pkt_duration;
+  frame->m_delay = (unsigned int)avframe->duration;
```

Comment:
This will most likely break everything that needs sync. pkt_duration is in seconds. the new duration is a different unit, see: https://ffmpeg.org/doxygen/6.0/structAVFrame.html#a91725a40000e348b0607adf7f577e646 vs. https://ffmpeg.org/doxygen/6.0/structAVFrame.html#ab90bd20a44c79c144045916b00c50e0d

Reply:
but ... that would most likely only break animated GIFS here :-)

Reply:
units of pts being 1/frame_rate?

Reply:
Yes - think so. But this has no impact on the tvheaded issues that is seen. Here it seems that the duration or the PTS of the packets AE checks are wrong. So it thinks stuff it adds to the audio sinks is smaller / larger or both.

Reply:
In which case the time unit is equivalent.

I think this change is correct based on a comparison of the ffmpeg source.  For example
https://ffmpeg.org/doxygen/5.0/ffmpeg_8c_source.html#l01176  vs.
https://ffmpeg.org/doxygen/6.0/ffmpeg_8c_source.html#l01279
The only change is pkt_duration being changed to duration.